ci/infra/testrunner/README.md

```diff
@@ -1,63 +1,213 @@
+
+* binpath: path to skuba binary
+* srcpath: path to skuba source. Used to locate other resources, like terraform configuration, and ssh keys.
+* verbosity: verbosity level for skuba command execution
 
 ```
 skuba:
-  binpath: "" # path to skuba binary
-  srcpath: "" # path to skuba source
+  binpath: "/usr/bin/"
+  srcpath: "/go/src/github.com/SUSE/skuba"
+  verbosity: 5
+```
+
+### Log
+
+Testrunner sends output to both a console and file logger handlers, configured using the following `log` variables:
+
+* file: name of the file used to send a copy of the log with verbosity `DEBUG`. This file is located under the `workspace` directory.
+* leve: debug verbosity level to console. Can be any of `DEBUG`, `INFO`, `WARNING`, `ERROR`. Defaults to `INFO`.
+* overwrite: boolean that indicates if the content of the log file must be overwritten (`True`) or log entries must be appended at the end of the file if it exists. Defaults to `False` (do not overwrite) 
+* quiet: boolean that indicates if `testrunner` will send any output to console (`False`) or not will execute silently (`True`). Quiet mode is useful when `testrunner` is used as a library. Defaults to `False`.
+
+Example:
+```
+log:
+  level: DEBUG
+```
+
+### Test
+
+* no_destroy: boolean that indicates if provisioned resources should be deleted when test ends. Defaults to `False`
+
+```
+no_destroy: True  #keep resources after test ends
+```
+
+## Environment Setup
+
+This section details how to setup `testrunner` 
+
+### Local setup
+
+Copy `vars.yaml` to `/path/to/myvars.yaml`, set the variables according to your environment and needs, and use the `--vars /path/to/myvars.yaml` CLI argument when running `testrunner`.
+
+#### Work Environment
+
+`testrunner` requires a working directory, which is specified in the `workspace` configuration parameter. The content of this directory can be erased or overwritten by `testrunner`. Be sure you create a directory to be used as workspace which is not located under your local working copy of the `skuba` project.
+
+
+```
+workspace: "/path/to/workspace"
+username:  "my-user-test"
+```
+
+
+#### skuba and platform
+
+Set the `skuba` and `terraform parameters depending on how you are testing `skuba`: 
+* If testing from local source:
+```
+  skuba:
+    srcpath: "/path/to/local/skuba/repo"
+    binpath: "path/to/go/bin/directory"
+```
+
+Be sure you don't specify the `terraform.tfdir` directory, so terraform configuration from the local `skuba` repo are used.
+
+
+* If testing from installed package
+
 ```
+  skuba:
+    binpath: "/usr/bin/"
 
-Anything else you can override is in `testrunner/vars.yaml` or can also be set as an environment variable.
+  terraform:
+    tfdir: "/path/to/terraform/files"
+```
+
+You use your `id_rsa` keys to connect to the cluster nodes, as the `shared_id` is not available.
```

Comment:
this is configurable now

Reply:
Not yet. PR is not merged

Reply:
I'll include an "update documentation" note in #593; there are a couple of places where the README will need tweaked after that change is applied.

Reply:
That PR is not yet merged and actually I asked for a change, so let's update the documentation as part of that PR.